platform-hub-auth-proxy/local/.env

```diff
@@ -9,3 +9,5 @@ export PHUB_AUTHPROXY_DATA_LOCAL
 export PHUB_AUTHPROXY_DATA_CONTAINER="/opt/data"
 
 export PHUB_AUTHPROXY_CONTAINER_NAME=phub-auth-proxy
+
+export KCPROXY_UPSTREAM_URL="http://10.200.10.1:8081"
```

Comment:
with a docker network/docker compose file this should be "http://<name-of-keycloack-service>:8081". No IP hardcoded